central/src/main/webapp/components/ModelView.jsx

```diff
@@ -217,7 +223,10 @@ export default function ModelView(props) {
 						<DynForm data={props.model.properties} />
 					</TabPanel>
 					<TabPanel value={index} index={2}>
-						<DynForm data={props.model.arguments} />
+						{data = props.model.arguments
```

Comment:
Should this be assigning to a variable data or just returning the ternary?

Reply:
I could change how it is written so that it would make more sense like:
```
{
    props.model.arguments //Is this a truthy value
      ? <DynForm data={props.model.arguments} />
      : <DynForm data={noArguments}/>
}
```
It supposed to check to see if the value is not null and then assign it to the variable data if it isn't